02_helper_functions/01_topics_mapping/setupTopics.m

```diff
@@ -84,3 +84,5 @@
     struct('topic_name', 'vehicle_status_flags', 'logged', false, 'num_instances', 0);
 topics.wind_estimate = ...
     struct('topic_name', 'wind_estimate', 'logged', false, 'num_instances', 0);
+topics.bat_mon = ...
```

Comment:
Can you put that in the right place in alphabetical order? This makes it easier to track which topics are already logged.